src/Application/UI/Presenter.php

```diff
@@ -475,7 +475,10 @@ public function findLayoutTemplateFile()
 	{
 		if ($this->layout === FALSE) {
 			return;
+		} elseif (file_exists($this->layout)) {
```

Comment:
I think having some check for `/` or `\`, before the IO call would be faster, than always doing a `file_exists`.